master/static/srv/notebook-entrypoint.sh

```diff
@@ -18,6 +18,30 @@ set -x
 test -f "${STARTUP_HOOK}" && source "${STARTUP_HOOK}"
 set +x
 
+if [ "$PACHD_ADDRESS" != "" ] && [ "$SKIP_PACHYDERM_INSTALL" != "true" ]; then
+    version=$(echo $(curl -s -X POST -H "Content-Type: application/json" pachyderm-proxy:80/api/versionpb_v2.API/GetVersion) | sed -n 's/.*"major":\([0-9]*\), "minor":\([0-9]*\), "micro":\([0-9]*\).*/\1.\2.\3/p')
+
+    pip install jupyterlab-pachyderm==$version
```

Comment:
should we bake this into envs images and set SKIP_PACHYDERM_INSTALL or do this? i think maybe @MikhailKardash is the right person to route this to?